File: 编译原理/TinyScript-Js/src/parser/ast/Variable.js
const Factor = require('./Factor')
const ASTNodeTypes = require('./ASTNodeTypes')

/**
 * 抽象语法树 - 变量节点
 */
class Variable extends Factor {
  /**
   * 
   * @param {ASTNode} _parent
   * @param {PeekTokenIterator} it
   */
  constructor(_parent, it) {
    super(_parent, it)
  }
}

module.exports = Variable

File: 编译原理/TinyScript/src/main/java/parser/ast/Variable.java
package parser.ast;

import parser.util.PeekTokenIterator;

/**
 * 抽象语法树 - 变量节点
 */
public class Variable extends Factor {
    public Variable(ASTNode _parent, PeekTokenIterator it) {
        super(_parent, it);
    }
}
